pairing/bn256/gfp.go

```diff
@@ -36,6 +39,29 @@ func newGFpFromBigInt(bigInt *big.Int) *gfP {
 	return out
 }
 
+func hashToBase(msg, dst []byte) *gfP {
```

Comment:
You are correct this function and a few others don't have use. They are used in a hash function that I think is already handled in other parts of the kyber implementation. Do you prefer that I bring it over as well or that I delete the unused functions ?

Reply:
I think this hash function might be of interest to us, if we can bring it over, yeah (and make sure to have tests to compare it with ours, because if ours is the same, then maybe we don't need it but I think it's not)

Reply:
I added the hash function and you were right it is not like ours